rules/create-appointment-for-flights/rule.md

```diff
@@ -1,31 +1,60 @@
-
-<!--endintro-->
-
-* Enter all of the flight details into the appointment - include the name and the flight number (⚠️as a minimum) plus the booking number and departure time in the subject. i.e. QF1234 – REF#102065 – 7:30pm. 
-* It’s also wise to include the exact location in the appointment, specifically the Terminal number. This will help make it as clear as possible, and you can see the relevant information without even opening the appointment. 
+---
+type: rule
+title: Do you know how to create an appointment in the Outlook calendar for flights?
+uri: create-appointment-for-flights
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+related: []
+redirects: []
+created: 2015-04-28T04:38:52.000Z
+archivedreason: null
+guid: c4aaf3fa-0a07-4293-b2cb-05569d67c8c8
+---
+
+This is how you should create an appointment in the Outlook calendar for flights:
+
+<!--endintro-->
+
+* Enter all of the flight details into the appointment - include the name and the flight number (⚠️as a minimum) plus the booking number and departure time in the subject. i.e. QF1234 – REF#102065 – 7:30pm. 
+* It’s also wise to include the exact location in the appointment, specifically the Terminal number. This will help make it as clear as possible, and you can see the relevant information without even opening the appointment. 
 * If someone is picking you up, include the arrival time and invite any relevant parties. E.g. The Boss's wife would like to know when he is flying out and coming home.
-* When possible, always include the terminal number in the location feild and if a non-standard terminal
+* When possible, always include the terminal number in the location field and if a non-standard terminal
 E.g. Jetstar flights highlight it with: "WARNING: Different terminal. Allow more time".
 * Make sure to include any unusual information about the flight
 E.g. "You are not flying with your preferred airline because xxx..." OR "As per our conversation, you are flying via Melbourne, because xxx..."
 * Always include the price of the ticket in the message
 * Do not forget to categorize the appointment as CONFIRMED RED so it cannot be overbooked
-* When possible, always include the terminal number in the location feild and if a non-standard terminal
-E.g. Jetstar flights highlight it with: "WARNING: Different terminal. Allow more time".
+* When possible, always include the terminal number in the location field and if a non-standard terminal
+E.g. Jetstar flights highlight it with: "WARNING: Different terminal. Allow more time".
 * Include the rule (Link) on the appointment. This will ensure you have quick access when double checking you have all the relevant info.
-
+
+Here is a template to follow
+
+::: email-template  
+|          |     |
+| -------- | --- |
+| To:      | {{Relevant parties}} |
+| Event Name: | ✈️ {{Flight Number}} - {{Booking number}} - {{Departure time}}  |

```

Comment:
Hey @bradystroud 

As per https://ssw.com.au/rules/placeholder-for-replaceable-text/

Placeholders should be UPPERCASE + have a space before and after the word. 

1. Please fix everywhere in your email template ;)